Gemfile

```diff
@@ -23,6 +23,7 @@ group :development do
   gem "voxpupuli-puppet-lint-plugins", '~> 5.0', require: false
   gem "facterdb", '~> 1.18',                     require: false
   gem "metadata-json-lint", '~> 3.0',            require: false
+  gem "puppetlabs_spec_helper", '~> 6.0',        require: false
```

Comment:
Is that pinned to an older version on purpose?

Reply:
yeah, even I wondered earlier but when I did backtraces I found it DevX team updated with older version.

Reply:
For ref : https://github.com/puppetlabs/pdk-templates/pull/546